crate/src/page/partial/content_control_panel.rs

```diff
@@ -5,7 +5,7 @@ use crate::{
     generated::css_classes::C, page::partial::image, Guide, Mode, Model, Msg,
     Urls,
 };
-use seed::{a, attrs, div, empty, prelude::*, span, style, C};
+use seed::{a, attrs, div, empty, prelude::*, span, style, C, IF};
```

Comment:
I think we can write standard `use seed::{prelude::*, *}` and force Clippy to ignore wildcard imports.